src/main/java/uk/ac/cam/cl/dtg/isaac/configuration/IsaacApplicationRegister.java

```diff
@@ -52,28 +50,18 @@
  * 
  */
 public class IsaacApplicationRegister extends Application {
-    private static final Logger log = LoggerFactory.getLogger(IsaacApplicationRegister.class);
-    private Set<Object> singletons;
+    private final Set<Object> singletons;
     
     public static Injector injector;
     
     /**
      * Default constructor.
      */
     public IsaacApplicationRegister() {
-        singletons = new HashSet<Object>();
+        singletons = new HashSet<>();
         SegueGuiceConfigurationModule segueGuiceConfigurationModule = new SegueGuiceConfigurationModule();
-        IsaacGuiceConfigurationModule isaacGuiceConfigurationModule = new IsaacGuiceConfigurationModule();
         
-        injector = Guice.createInjector(isaacGuiceConfigurationModule, segueGuiceConfigurationModule);
-        
-        SegueConfigurationModule segueConfigurationModule = injector.getInstance(SegueConfigurationModule.class);
-        ContentMapper mapper = injector.getInstance(ContentMapper.class);
-        if (segueConfigurationModule != null) {
-            // register the isaac specific data types.
-            log.info("Registering isaac specific datatypes with the segue content mapper.");
-            mapper.registerJsonTypes(segueConfigurationModule.getContentDataTransferObjectMap());
-        }
+        injector = Guice.createInjector(segueGuiceConfigurationModule);
```

Comment:
There is now only one Guice configuration module, and only one ContentMapper configuration (covering both packages now), so all of this is unnecessary.